lib/common/compiler.h

```diff
@@ -278,6 +278,8 @@
 *  Alignment check
 *****************************************************************/
 
+#define ZSTD_IS_POWER_2(a) (((a) & ((a)-1)) == 0)
```

Comment:
nit: We don't need this for compile time constants like array bounds, could we make this a function `ZSTD_isPow2()`?

Reply:
done